networks.json

```diff
@@ -106,20 +106,34 @@
     ]
   },
   {
-    "name": "MEETONE Mainnet",
-    "description": "MEETONE Mainnet",
-    "owner": "MEETONE",
-    "chainId": "cfe6486a83bad4962f232d48003b1824ab5665c36778141034d75e57b956e422",
+    "name": "Freedom First Mainnet",
```

Comment:
Should be called `Freedom First Testnet`

Seems like there's no identifiable block producers

```
cleos -u https://dconnect.live system listproducers

Producer      Producer key                                              Url                                                         Scaled votes
evo           EOS5ap3iQbJY89hGi75tJTa2fh6iv3J2wTk5UkeHyTrJRTucBKy9e                                                                 0.1135
atechshaman   EOS8iK6Jq73YFrfWWebkhbfj4f3GdVokRHJAwcxJSLwz5sQb8BZba                                                                 0.1135
phil          EOS7UnjhXYfUnTTrpcFP9GhMF7ubcd2sSJqBNSCBii9otguUK8zTJ                                                                 0.1105
webbed        EOS7BECyXpgjXeLaFH6LrsKmZMQApy8TDJPC7B3tU6eHfgor8z69q                                                                 0.0927
freedomfirst  EOS65eQ8DtWwgyBXeYvmpjPrHo49ohrWuQjcAcWQ4tKv13AyEXiby                                                                 0.0783
youroneangel  EOS549KADkBUMJceUUyaW6wQzZ84eEgf4Nr4iwdL4d2Xda7FDKL8s     www.google.com                                              0.0767
occultcorner  EOS77UdptM6zoiGtMGRbzBnPMZ81AcMq1qozb7F3Ycv85eG6tEfcZ                                                                 0.0767
koh           EOS5K7EkWPhgETBkPtJ3KFB1P1HBb2yseGiYCQQzmFcT9Y8vQRURD                                                                 0.0751
cheeseburger  EOS5RejDrUr7hyo2hHEtBPBScfirV1bs8YrAqXe3riCxsCjNvW4d1                                                                 0.0749
moonshine     EOS84tXCUNRVAjFBZCXFjfopKgNpk4jNNZnxwqCxssVbCC11NTZrq                                                                 0.0746
supermeatboy  EOS6AjN55SaQwtLZUmTN7NU6Gp98WhRMwLNaFkxcchQ87mF1Bio6U                                                                 0.0736
zeshanyousaf  EOS8gYqrxJgr4C74JZRchZ1byfkQYPiNoy1zbYPJp2TGWg6Apc5An                                                                 0.0399
```